src/main/java/nl/stil4m/transmission/api/domain/AddedTorrentInfo.java

```diff
@@ -6,14 +6,31 @@
 public class AddedTorrentInfo {
 
 
-    private final TorrentInfo torrentAdded;
+    private final TorrentInfo torrentInfo;
+
+    private final boolean isDuplicate;
 
     @JsonCreator
-    public AddedTorrentInfo(@JsonProperty("torrent-added") TorrentInfo torrentAdded) {
-        this.torrentAdded = torrentAdded;
+    public AddedTorrentInfo(@JsonProperty("torrent-added") TorrentInfo torrentAdded,
+                            @JsonProperty("torrent-duplicate") TorrentInfo torrentDuplicate) {
+        TorrentInfo newTorrentInfo = null;
+        if (torrentAdded != null) {
+            newTorrentInfo = torrentAdded;
+        }
+        if (torrentDuplicate != null) {
+            isDuplicate = true;
+            newTorrentInfo = torrentDuplicate;
+        } else {
+            isDuplicate = false;
+        }
+        this.torrentInfo = newTorrentInfo;
+    }
+
+    public TorrentInfo getTorrentInfo() {
+        return torrentInfo;
     }
 
-    public TorrentInfo getTorrentAdded() {
-        return torrentAdded;
+    public boolean isDuplicate() {
+        return isDuplicate;
```

Comment:
I see what you do here, but I feel that it is a bit too complex.

Why not create two fields: `torrentAdded` and `torrentDuplicate` and create 4 accessors: 

* `isDuplicate` (containing the logic if `this.torrentDuplicate != null`)
* `getTorrentAdded`
* `getTorrentDuplicate`
* `getTorrentInfo` which will return either `torrentAdded` if it is not null or `torrentDuplicate`.